alacritty/src/window.rs

```diff
@@ -345,13 +345,6 @@ impl Window {
     }
 
     pub fn set_fullscreen(&mut self, fullscreen: bool) {
-        #[cfg(macos)]
-        {
-            if self.window().simple_fullscreen() {
-                return;
-            }
-        }
-
```

Comment:
The same should be done for `set_simple_fullscreen`.

Reply:
Oh I was looking for a `cfg(..)` inside set_simple_fullscreen :sweat_smile:  sorry....